integration-tests/int-test-runner.sh

```diff
@@ -4,6 +4,11 @@ export STATUS=$?
 
 if [ $STATUS -eq 1 ]
 then
-  curl -X POST --data-urlencode 'payload={"text": "Media Atom Maker integration tests have failed on CODE"}' ${SLACK_URL}
+  if [ "$INT_TEST_TARGET" == "PROD" ]
+  then
+    curl -X POST --data-urlencode 'payload={"text": "<!here> Media Atom Maker integration tests have failed on PRODUCTION"}' ${SLACK_URL}
```

Comment:
I'll add in link to the project. Could do a test result file, but considering these are 'monitoring' tests, they're running every 15 mins, so could end up filling up team city with artefacts quite quickly. From the project page, can get the failing build log really quickly